source/includes/azure/_disks.md

```diff
@@ -108,7 +108,7 @@ Attributes | &nbsp;
 `name`<br/>*string* | The display name of the disk
 `type`<br/>*string* | The managed disk type. Can be one of the following: `premium_lrs` _(Premium SSD)_, `standard_lrs` _(Standard HDD)_, `standardssd_lrs` _(Standard SSD)_ or `ultrassd_lrs` _(Ultra SSD)_
 `region`<br/>*string* | The regional location in which the disk was instantiated
-`instanceId`<br/>*string* | The id of the instance to which the disk is attached to. This is a canonized id from azure which is the form of `/subscriptions/${subscriptionid}/resourceGroups/${resourcegroup}/providers/Microsoft.Compute/virtualMachines/${instanceName}` 
+`instanceId`<br/>*string* | The id of the instance to which the disk is attached to. This is a canonized id from azure which is the form of `/subscriptions/${subscriptionid}/resourceGroups/${resourcegroup}/providers/Microsoft.Compute/virtualMachines/${instanceName}`
```

Comment:
Is this still the case? Didn't @Shabirmean change it to use the name?

Reply:
The PR is in draft. Had to halt it since I will have to verify all the entities. So banked it until all the current operation PRs are merged. And then will have to review the docs as well as part of that story